sdk/inc/azure/core/az_span.h

```diff
@@ -34,21 +34,23 @@ typedef struct
   struct
   {
     uint8_t* ptr;
-    int32_t size; ///< size must be &ge; 0
+    int32_t size; // size must be >= 0
   } _internal;
 } az_span;
 
 /********************************  SPAN GETTERS */
 
 /**
  * @brief Returns the #az_span byte buffer's starting memory address.
- *
+ * @param[in] span An #az_span to get the buffer's starting memory address of.
+ * @return Starting memory address of \p span buffer.
```

Comment:
Is having a `@return` required? I feel like repeating the same information three times would unnecessarily clutter up the docs and force users to read more.

Reply:
I'd leave it. If we are commenting, let's stick to some set of things that we always put there. Maybe you are right, maybe it is subjective. If I was just writing docs from myself, I'd also have some things writtern differently, or some left not documented. But what's obvious for me, may not be obvious for others.